templates/iqe-trigger-integration.yml

```
@@ -124,13 +124,16 @@ parameters:
   from: "[a-z0-9]{6}"
 - name: IQE_IMAGE
   description: "container image path for the iqe plugin"
-  value: quay.io/cloudservices/iqe-tests:hms-integration
+  value: quay.io/cloudservices/iqe-tests
```

```suggestion
  value: quay.io/cloudservices/iqe-tests:insights-experiences
```
I guess let's add this here, that should make the linter happy and it makes sense to me to have a tag there.